core/src/main/java/org/mskcc/cbio/portal/scripts/ImportStructuralVariantData.java

```diff
@@ -160,7 +161,7 @@ private CanonicalGene setCanonicalGene(long siteEntrezGeneId, String siteHugoSym
         }
 
         // If no gene can be found based on Entrez Gene ID, try Symbol.
-        if (siteCanonicalGene == null) {
+        if (siteCanonicalGene == null && !TabDelimitedFileUtil.NA_STRING.equals(siteHugoSymbol)) {
```

Comment:
this is an bug fix .. when we noticed records showing up because "NA" is a valid gene symbol alias (reported by curators)